CHANGELOG_PENDING.md

```diff
@@ -23,6 +23,8 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### FEATURES
 
+- [libs/math] \#5665 Make fractions unsigned integers
```

Comment:
```suggestion
- [libs/math] \#5665 Make fractions unsigned integers (uint64) (@cmwaters)
```
Not sure this is a feature. May be better under Improvements

Reply:
Yeah I had this small mental debate because it's not really an improvement either

Reply:
fractions can be negative - it's just it makes sense in the context of Tendermint that fractions are positive only